### ISSUE TITLE
Interactive window is unavailable on Express SKUs

### ISSUE THREAD

**Issue by zooba** [dotnet/roslyn#3839]
The interactive window fails to load on Express SKUs because the VSCT is not being loaded.

This appears at runtime as an `E_INVALIDARG` response from `toolbarHost.AddToolbar` when called from `VsInteractiveWindow.OnToolWindowCreated()`.

Without attempting to open an interactive window, you can observe that none of the commands from the VSCT appear under Tools, Options, Keyboard. On Community+, you can see all the commands here.

(Tested with Express for Web. Originally reported as https://github.com/Microsoft/PTVS/issues/544)

**Comment by zooba** [comment]
I suspect this is somehow due to Express SKUs excluding unnecessary commands, which is also including the VSCT embedded in `Microsoft.VisualStudio.VsInteractiveWindow.dll`. I'm going to test whether we can work around this by providing a "matching" VSCT ourselves.

**Comment by zooba** [comment]
It looks like we can work around this issue by including another VSPackage that provides an exact copy of the InteractiveWindow.vsct file, so I'm not sure what's going on, but it doesn't seem to be related to the VSCT content.

**Comment by ManishJayaswal** [comment]
@DinoV @zooba - is this while trying to open python interactive window in express SKU? I am not sure what the expected behavior should be, or what the issue is. @DinoV  may have more context.

**Comment by zooba** [comment]
The expected behavior is that the window opens. The actual behavior is that a message box appears saying E_INVALIDARG (and with a debugger, I got the stack traces in the linked issue above).

The problem is that the VSCT included in the core interactive window is not being merged into the CTM file for VS Express. The workaround I added to PTVS is nothing more than a VSPackage with a copy of the VSCT file and it loads fine. I wasn't able to determine why the core VSCT is not being loaded.
